packages/notification-services-controller/src/NotificationServicesController/NotificationServicesController.test.ts

```diff
@@ -55,6 +58,11 @@ const featureAnnouncementsEnv = {
   platform: 'extension' as const,
 };
 
+// Testing util to clean up verbose logs when testing errors
+const mockErrorLog = () =>
+  jest.spyOn(log, 'error').mockImplementation(jest.fn());
+const mockWarnLog = () => jest.spyOn(log, 'warn').mockImplementation(jest.fn());
```

Comment:
`test:verbose` is a bit too messy since we are testing a lot of failure cases.
This cleans up the logs only for cases we know we are failing on.